cfg/genesis.json

```diff
@@ -62,7 +62,8 @@
           "3C43D9A973AA4443EF3FC38E42DD306160FBFFDAB901CD8BAA15D09F2597EB87",
           "0285B7E5E08E1A8E4C15636F0591D87F73CB6A7B6452A932AD72BBC8E5D1CBE3",
           "6E739F4F8B07BED29FC9FF440DA3C301CD14A180DF45819F658FEC2F7DE31427",
-          "36799EA497B1369B170805C078AEFE6188345F9B3E324C21E9CA3FF574E3C3D6"
+          "36799EA497B1369B170805C078AEFE6188345F9B3E324C21E9CA3FF574E3C3D6",
+          "DD4F86291F142A20761B32B4D0CE4291F86CA33F0B46F0D04171482FBA52E536"
```

Comment:
Maybe this shouldn't be changed.

@RichardAH

Reply:
Yeah, maybe not
We are targeting the jshooks branch though

Reply:
I did note that the xrpld-network-gen project uses a --ledgerfile with no amendments at all, and must rely on some other way of enabling them:
https://github.com/Transia-RnD/xrpld-network-gen/blob/main/xrpld_netgen/genesis.xahau.json